gemfiles/rails_6.1.gemfile

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in lograge.gemspec
+gemspec path: '..'
+
+group :test do
+  gem 'actionpack', '~> 6.1.0'
+  gem 'activerecord', '~> 6.1.0'
+  # logstash does not release any gems on rubygems, but they have two gemspecs within their repo.
+  # Using the tag is an attempt of having a stable version to test against where we can ensure that
+  # we test against the correct code.
+  gem 'logstash-event', git: 'https://github.com/elastic/logstash', tag: 'v1.5.4'
```

Comment:
@iloveitaly what is the rational behind using this version ?

Reply:
IIRC (it's been a while), some things changed in the Logstash codebase that removed a class that's depended on in one of Lograge's adapters.